docs/geometry.rst

```diff
@@ -1,78 +1,96 @@
 Geometry
 ========
 
-The ``pygeos.Geometry`` class is the central datatype in PyGEOS. 
-An instance of ``Geometry`` is a container of the actual GEOSGeometry object.
-The Geometry object keeps track of the underlying GEOSGeometry and
-lets the python garbage collector free its memory when it is not
-used anymore.
+Shapely geometry classes, such as ``shapely.Point``, are the central data types
+in Shapely.  Each geometry class extends the ``shapely.Geometry`` base class,
+which is a container of the underlying GEOS geometry object, to provide geometry
+type-specific attributes and behavior.  The ``Geometry`` object keeps track of
+the underlying GEOS geometry and lets the python garbage collector free its
+memory when it is not used anymore.
 
 Geometry objects are immutable. This means that after constructed, they cannot
-be changed in place. Every PyGEOS operation will result in a new object being returned.
+be changed in place. Every Shapely operation will result in a new object being
+returned.
+
 
 Construction
 ~~~~~~~~~~~~
 
-For convenience, the ``Geometry`` class can be constructed with a WKT (Well-Known Text)
-or WKB (Well-Known Binary) representation of a geometry:
+Geometries can be constructed directly using Shapely geometry classes:
```

Comment:
seemed like we needed a canonical example here of using the classes, rather than starting with WKT / WKB examples.